test/index.ts

```diff
@@ -105,7 +74,8 @@ describe('Mutation Test', () => {
       name: correctInputUser.name,
       birthDate: correctInputUser.birthDate,
     });
-    expect(createUseMutation.data.data.createUser.id).to.exist;
+
+    expect(mutationReturn.id).to.exist;
```

Comment:
Acho que seria uma boa testar ao menos o formato do id. Se for um inteiro, checar se é inteiro > 0